testsuite/features/build_validation/create_bootstrap_repositories/create_bootstrap_repository.template

```diff
@@ -8,4 +8,4 @@ As the system administrator
 I create all bootstrap repos with --with-custom-channels option
 
   Scenario: Create the bootstrap repository for <client>
-    When I create the bootstrap repository for "<client>" on the server
+    When I create the bootstrap repository for "<client>" on the server <flush>
```

Comment:
I don't think  this step definition will match in the case we add 'without flushing'.

```ruby
When(/^I create the bootstrap repository for "([^"]*)" on the server$/) do |host|
```

Is that intended because what's needed for s390 is already created by the bootstrap for a standard minion ?